frontend/server/src/DAO/Problems.php

```diff
@@ -537,6 +537,40 @@ public static function getProblemsSolvedCount(\OmegaUp\DAO\VO\Identities $identi
         return $problems;
     }
 
+    /**
+     * Returns the list of problems created by a certain identity
+     * @param int $identityId
+     * @return \OmegaUp\DAO\VO\Problems[]
+     */
+    final public static function getProblemsCreatedByIdentity(
```

Comment:
nit:

```suggestion
    final public static function getPublicProblemsCreatedByIdentity(
```

así es más obvio que sólo son los problemas públicos.